components/NavButtons/FAQ.tsx

```diff
@@ -9,3 +9,4 @@ export const FAQButton = () => (
       </button>
     </Link>
 );
+// Delete this 
```

Comment:
Why not just replace this component instead of replacing it inline in Header? Feels cleaner

Reply:
I guess me thinking was why have a component in the a component? Would this result in performance loss?

Reply:
re: performance loss - I don't think so - but if you can find something explaining why it does I'd be open to changing my ways. 

The reason I put it in its own component in the first place is that I use it in 2 spots. 
if you replace the component, you'll automatically update both spots.